src/AppBundle/Entity/Report/MoneyTransaction.php

```diff
@@ -93,7 +93,8 @@ class MoneyTransaction implements MoneyTransactionInterface
 
         ['deputy-security-bond', false, 'fees', 'out'],
         ['opg-fees', false, 'fees', 'out'],
-        ['professional-fees-eg-solicitor-accountant', true, 'fees', 'out'],
+        ['professional-fees-eg-solicitor-accountant-lay', true, 'fees', 'out'],
```

Comment:
Id be tempted to keep existing string as is and just introduce a new one. Im thinking of existing data on prod

Reply:
👍